src/Badge.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Badge extends React.Component {
+  render() {
+    let classes = ['badge'];
+    if (this.props.text) {
+      classes.push(this.props.text);
```

Comment:
You don't have to close this PR, if you just push your change to your branch it will show here.